migrate/labels.yml

```diff
@@ -0,0 +1,118 @@
+# Repository labels to be synced across the entire org.
+# Every item must have a name, color, and description.
+
+# Notes:
+# * Repositories may define their own labels as long as they don't
+#   conflict with what's listed here. If it turns out a repository-specific
+#   label would be useful to have in the entire organization, we can always add
+#   it here post facto.
+# * We have added these based on need. They are not all-encompassing. If something
+#   seems missing, feel free to open a PR.
+# * Whenever a repo has a matching label, that label will be updated. If there
+#   is no matching label, the label will be created anew. "Matching" is determined
+#   by comparing the label's lowercased name, with all puncutation, emoji, and special
+#   chars removed.
+# * For picking hex colors, you might use: https://www.htmlcolor-picker.com/
+#   Describe your color with a comment so it's easier to review.
+
+
+### LABELS INDICATING HOW ISSUES CAN BE ENAGAGED WITH.
+
+- name: "good first issue"
+  color: "42dd35"  # friendly lime green
+  description: "A good task for a newcomer to start with"
+
+- name: "help wanted"
+  color: "54976d"  # fenway green
+  description: "Ready to be picked up by anyone in the community"
```

Comment:
Who will be adding this label to issues?  Sort of by default, all issues are "help wanted".  Is there some specific kind of ready this is meant to indicate other than it's an open issue?

Reply:
@nedbat To me, the `help wanted` label is meant to answer the question "I'd like to help with this group/project, what should I work on next?" Consider it a superset of `good first issue`.

Open issues that _wouldn't_ warrant a `help wanted` labels are:
* a "draft" issue; one that hasn't been refined well, maybe something you wrote up quickly with the intent of fleshing out later, like [this one](https://github.com/openedx/edx-platform/issues/31768).
* issues that require special permissions, like Axim requests, or certain WG tickets [like this one](https://github.com/openedx/wg-security/issues/29).
* ones that require special context [like this one](https://github.com/openedx/openedx-k8s-harmony/issues/28).
* issues with a high technical barrier to entry. 
* issues that are part of an actively  funded project (ie, we are already paying someone to do it).
* issues that already have intended assignees, like [this one](https://github.com/openedx/wg-build-test-release/issues/279).

> Who will be adding this label to issues? 

Anyone! I'll be using it for my working group, and I'll encourage other WG leads to as well.

Reply:
This sort of guidance will be good to have somewhere.  This file is not discoverable, but I'm not sure where else to put it.

Reply:
OK, maybe this is something we could add to maintainer playbooks or contribution guides on docs.openedx.org.

That said, I am hoping that the labels can be as self-explanatory as possible. We can't assume potential new contributors will be reading docs.openedx.org, so the more obvious we can make ourselves to the outside world with the tools GitHub gives us, the better.

Is it not reasonably clear what `help wanted` means?

Reply:
My only concern about "help wanted" is the interpretation that if the label isn't on a PR, then we don't want or need help with it.

My suggestion about putting your guidelines somewhere was not so that new contributors would find them, but so that we who are labelling issues would have a common understanding of the labels.

Reply:
> My suggestion about putting your guidelines somewhere was not so that new contributors would find them, but so that we who are labelling issues would have a common understanding of the labels.

That makes sense. I think the maintainership playbooks would be a good home for these docs, then.